jen-jin/moment
# Reflection First Iteration

Targets: [Reflection Ticket](https://speaku.atlassian.net/browse/FYDP-50)

Details
- Created the UI for the reflection screen
- No tests written yet

Testing
- npm run/yarn start
- Navigate to /reflection for your localhost
- Navigate through reflection questions with the back and next buttons
- Confirm step number updated correctly, able to type in textboxes, able to select options
- Verify URL remains the same through the question
